TF_Service_dotNet/TouchFree_Service/Connection/TouchFreeServiceTypes.cs

```diff
@@ -25,7 +28,28 @@ internal enum Compatibility
     {
         COMPATIBLE,
         SERVICE_OUTDATED,
-        CLIENT_OUTDATED
+        CLIENT_OUTDATED,
+        SERVICE_OUTDATED_WARNING,
+        CLIENT_OUTDATED_WARNING
+    }
+
+    public enum CameraState
```

Comment:
Was thinking about this, currently if the tracking Service isn't connected, we'll report Camera NOT_CONNECTED even if it is physically, as we can't get at the information about the hardware without the Service. Was going to suggest a third entry here along the lines of "NO_SERVICE" to indicate that but then I realised at that point we may as well combine this into one enum, as we can only (reliably) report 

* Everything fine
* No Camera
* No Service

Do you think it's worth the consolidation here as the scenario of "No Service, Camera" will never actually be possible?

Reply:
I think that makes sense. I'll start making the changes now

Reply:
Done!